### PR TITLE
Bug 1713303 - Provide link to next/previous alert

### PR DESCRIPTION
**Pull request by esanuandra**

### CODE REVIEW

**Comment by esanuandra** [comment]
next_prev_alert_button_2_bottom_table
next_prev_alert_button_3_notes
next_prev_alert_button_action_panel
next_prev_alert_button_notes_reset
next_prev_alert_button_1

File: ui/perfherder/alerts/AlertsViewControls.jsx [hunk 1]
```diff
@@ -35,6 +59,34 @@ export default class AlertsViewControls extends React.Component {
...
+      prev: 'prev',
+      next: 'next',
+    };
+
+    if (type === scrollTypes.next) {
+      currentAlert = currentAlert === alertsLength - 1 ? 0 : currentAlert + 1;
+    }
+
+    if (type === scrollTypes.prev) {
+      currentAlert = currentAlert <= 0 ? alertsLength - 1 : currentAlert - 1;
```
Comment (airimovici) [inline ui/perfherder/alerts/AlertsViewControls.jsx:84]: I think the `Next` button should be disabled when you reach the last alert summary.
Same for first alert summary and `Previous` button.

Right now it goes in a circle.

**Comment by airimovici** [comment]
When there is no Notes for a table or an alert summary is not selected, there should be a background behind the buttons

**Review by beatrice-acasandrei — APPROVED** [review]
LGTM

**Review by sarah-clements — APPROVED** [review]

**Comment by esanuandra** [comment]
Hey, @sarah-clements ! Thank you for the review. Could you please merge this into master? @airimovici is PTO this week.

**Pull request merged**